examples/grpc/setup.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const { NodeTracer } = require('@opentelemetry/node-sdk');
+const { SimpleSpanProcessor } = require('@opentelemetry/basic-tracer');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const EXPORTER = process.env.EXPORTER || '';
+
+function setupTracerAndExporters(service) {
+  let exporter;
+  const options = {
+    serviceName: service
+  };
+  const tracer = new NodeTracer({
+    plugins: {
+      grpc: {
+        enabled: true,
+        // if it can't find the module, put the absolute path since the packages are not published yet
+        path: '@opentelemetry/plugin-grpc',
+        ignoreOutgoingUrls: [/spans/]
```

Comment:
Do you use it ?

Reply:
Also you should add this in http otherwise you will get infinity spans with Zipkin
#332

Reply:
Nope, removed